docs/sources/ingest-and-analyze-profile-data/profile-cli.md

```diff
@@ -216,19 +241,30 @@ You can use the `profilecli query merge` command to retrieve a merged (aggregate
      export PROFILECLI_PASSWORD=<password>
 
      profilecli query merge \
-         --query=<label_name>=<label_value>
          --profile-type=<profile_type>
+         --query='{<label_name>="<label_value>"'
      ```
 
-   - Modify the placeholders (`<pyroscope_server_url>`, `<username>`, etc.) with your actual values.
-
    - Example command:
      ```bash
      export PROFILECLI_URL=https://profiles-prod-001.grafana.net
      export PROFILECLI_USERNAME=my_username
      export PROFILECLI_PASSWORD=my_password
 
      profilecli query merge \
-         --query='{service_name="my_application_name"}' \
          --profile-type=memory:inuse_space:bytes:space:bytes
+         --query='{service_name="my_application_name"}' \
```

Comment:
can we explicitly write out the defaults so that people copy-pasting are aware of how and what format to change the time frame for something like this ?

Reply:
also is this really "merging" or just querying? 

for example if I query data from 12:00 to 12:45 and I merge data from 12:00-12:45 isn't that the same thing?

Reply:
> can we explicitly write out the defaults so that people copy-pasting are aware of how and what format to change the time frame for something like this ?

Done!

> also is this really "merging" or just querying?
> 
> for example if I query data from 12:00 to 12:45 and I merge data from 12:00-12:45 isn't that the same thing?

We call it "merge" throughout the codebase because we are (potentially) merging data from multiple profiles into one. I agree that it is a confusing word, merge is often used for mutating operations (e.g., `git merge`).

I don't think we can call it just `query` here since we have two subcommands (`merge` and `series`) but we could call it `query profile` potentially (with the risk of confusion about whether it returns data from one or more profiles). I'll leave it as it is for now though to avoid a breaking change at this time.